ports/stm32/boards/NUCLEO_WB55/pins.csv

```diff
@@ -1,49 +1,40 @@
-,PA0
-,PA1
-,PA2
-,PA3
-,PA4
-,PA5
-,PA6
-,PA7
-,PA8
-,PA9
-,PA10
-,PA11
-,PA12
 ,PA13
 ,PA14
-,PA15
-,PB0
-,PB1
 ,PB2
 ,PB3
 ,PB4
-,PB5
 ,PB6
 ,PB7
-,PB8
-,PB9
 ,PB10
 ,PB11
 ,PB12
 ,PB13
 ,PB14
 ,PB15
-,PC0
-,PC1
-,PC2
-,PC3
-,PC4
-,PC5
```

Comment:
It looks like PC5 was completely removed.  Was that intentional?

Reply:
It's just an error of may part